src/app/Rules/CreateRule.tsx

```diff
@@ -414,11 +414,11 @@ export const CreateRuleForm: React.FC<CreateRuleFormProps> = (_props) => {
         fieldId="rule-matchexpr"
         helperText={
           evaluating
-            ? 'Evaluating match expression...'
+            ? 'Evaluating Match Expression...'
             : formData.matchExpressionValid === ValidatedOptions.warning
-            ? `Warning: Match expression matches no targets.`
+            ? `Warning: Match Expression matches no targets.`
             : `
-  Enter a match expression. This is a Java-like code snippet that is evaluated against each target
+  Enter a Match Expression. This is a Java-like code snippet that is evaluated against each target
```

Comment:
Also maybe for now or maybe for a follow-up, but I think we can (finally!) replace this "Java-like code snippet" cop-out description with a reference to https://github.com/google/cel-spec . Back when I wrote this description the actual implementation was a restricted JavaScript parser, but the long term goal was to replace that with either a purpose-built parser with similar syntax, or with an embedded scripting/evaluation language with similar syntax - and CEL is that embedded evaluation language.

Reply:
Ok, I will look into this.

Reply:
Opened a new issue for this [https://github.com/cryostatio/cryostat-web/issues/1263](https://github.com/cryostatio/cryostat-web/issues/1263)